### PR TITLE
Removed unneeded spaces

### PR DESCRIPTION
**Pull request by WyattMufson**
Don't need the spaces for `[INFO]` and `[WARN]`

### CODE REVIEW

File: common/log/log.go [hunk 1]
```diff
@@ -59,8 +59,8 @@ const (
 var (
 	levels = map[int]string{
 		DebugLog: Color(Green, "[DEBUG]"),
-		InfoLog:  Color(Cyan, "[INFO ]"),
-		WarnLog:  Color(Yellow, "[WARN ]"),
+		InfoLog:  Color(Cyan, "[INFO]"),
```
Comment (laizy) [inline common/log/log.go:62]: this is intended to make the ouput aligned，since `debug`,'error` 'fatal`,and `trace` are 5 letters.

Comment (WyattMufson) [reply common/log/log.go:62]: Is alignment required?

If so, would maybe adding a `.` or `!` be better than a space?

Comment (WyattMufson) [reply common/log/log.go:62]: It just looked a little off when I saw it

**Comment by WyattMufson** [comment]
I think it is fine as is

**Pull request closed without merging**